packages/gatsby/src/utils/adapter/init.ts

```diff
@@ -4,13 +4,14 @@ import { createRequireFromPath } from "gatsby-core-utils/create-require-from-pat
 import { join } from "path"
 import { emptyDir, ensureDir, outputJson } from "fs-extra"
 import execa, { Options as ExecaOptions } from "execa"
-import { version as gatsbyVersion } from "gatsby/package.json"
+import { version as gatsbyVersionFromPackageJson } from "gatsby/package.json"
 import { satisfies } from "semver"
 import type { AdapterInit } from "./types"
 import { preferDefault } from "../../bootstrap/prefer-default"
 import { getLatestAdapters } from "../get-latest-gatsby-files"
 
-const getAdaptersCacheDir = (): string => join(process.cwd(), `.cache/adapters`)
+export const getAdaptersCacheDir = (): string =>
+  join(process.cwd(), `.cache/adapters`)
```

Comment:
Small nit (not-blocking), could put the path in a constant

Reply:
It probably could, tho I vaguely remember a PR long time ago that moved similar path constructor from constant to function for some usecase where CWD changed after modules were imported.

It also was already a function and only change I did was to export it (to use in tests), so IMO there are is not enough benefits to mess with that (cost of running `path.join` couple of times vs just once is something you can't even measure as variance will be much greater than perf difference), so I will just keep it as-is